client/src/components/Account/ResetPassword.js

```diff
@@ -38,7 +38,10 @@ const styles = (theme) => ({
 const validationSchema = Yup.object().shape({
   token: Yup.string().required("Token is required"),
   password: Yup.string()
-    .min(8, "Password must be 8 characters at minimum")
+    .matches(
+      /^.*(?=.{8,})((?=.*[!@#$%^&*()\-_=+{};:,<.>]){1})(?=.*\d)((?=.*[a-z]){1})((?=.*[A-Z]){1}).*$/,
+      "Password must contain at least 8 characters, one uppercase, one number and one special case character"
```

Comment:
Same with this error message 👌

Reply:
I see there are similar validation message used in more than one component, should I also store them. For e.g: "Password is required", "Email is required"

Reply:
I see there are similar validation message used in more than one component, should I also store them? 🤔  For e.g: "Password is required", "Email is required"